ouroboros-consensus/docs/Onboarding.md

```diff
@@ -3,10 +3,116 @@
 The content in this file should help orient a new contributor to the Consensus
 Layer: its fundamental purpose and its neighboring components.
 
+The HR will run separate "on-boarding" that will help you to get to know
+the company better. The purpose of this document is to make you feel comfortable
+and productive in the Consensus team.
+
+Welcome, we are thrilled to have you here :)!
+
 Please open PRs to add particularly useful content here. Though we would like
 the file to remain as approachable as possible, therefore small and
 narrowly-scoped.
 
+## The team
```

Comment:
We'll move the IOHK stuff to a separate file now. Maybe to an entirely different repo later, if some repository becomes an obvious choice for that.

Reply:
PR #2903 added this file as `ouroboros-consensus/docs/IOHKOnboarding.md`